File: src/main/scala/im/tox/toktok/app/File.scala
package im.tox.toktok.app

class File(name: String, date: String) {

  val fileName: String = name
  val fileDate: String = date

  def getName(): String = {
    return fileName
  }

  def getDate(): String = {
    return fileDate
  }

}

object File {
  val file = new File("Movie_2015-02-01.mp4", "Downloaded 2015-05-21")
}

File: src/main/java/im/tox/toktok/app/domain/File.java
package im.tox.toktok.app.domain;

public final class File {
    public static final File file = new File("Movie_2015-02-01.mp4", "Downloaded 2015-05-21");

    public final String name;
    public final String date;

    public File(
            String name,
            String date
    ) {
        this.name = name;
        this.date = date;
    }
}
